pkg/neonvm/controllers/vmmigration_controller.go

```diff
@@ -309,11 +309,16 @@ func (r *VirtualMachineMigrationReconciler) Reconcile(ctx context.Context, req c
 			migration.Status.SourcePodIP = vm.Status.PodIP
 			migration.Status.TargetPodIP = targetRunner.Status.PodIP
 
-			// do hotplugCPU in targetRunner before migration
+			// do cpu hot plug in targetRunner before migration
+			// in case of QMP mode, we need to sync CPUs before migration
+			// in case of Sysfs mode, we need to sync CPUs during migration
```

Comment:
> in case of Sysfs mode, we need to sync CPUs during migration

Why? I would have thought it just works, because all of the CPUs are already hotplugged in both places?

Reply:
Mostly because at that moment machine is not booted yet, so it's kind chicken-and-egg thing.